flexget/api/core/authentication.py

```diff
@@ -18,42 +21,37 @@
 
 @login_manager.request_loader
 @with_session
-def load_user_from_request(request, session=None):
+def load_user_from_request(request: Request, session: Session = None) -> Optional[User]:
     auth_value = request.headers.get('Authorization')
 
     if not auth_value:
-        return
+        return None
 
     # Login using api key
     if auth_value.startswith('Token'):
-        try:
+        with suppress(TypeError, ValueError):
             token = auth_value.replace('Token ', '', 1)
             return session.query(User).filter(User.token == token).first()
-        except (TypeError, ValueError):
-            pass
 
     # Login using basic auth
     if auth_value.startswith('Basic'):
-        try:
-            credentials = base64.b64decode(auth_value.replace('Basic ', '', 1))
+        with suppress(TypeError, ValueError, UnicodeDecodeError):
+            credentials = base64.b64decode(auth_value.replace('Basic ', '', 1)).decode()
             username, password = credentials.split(':')
             user = session.query(User).filter(User.name == username).first()
             if user and user.password and check_password_hash(user.password, password):
                 return user
-            else:
-                return None
-        except (TypeError, ValueError):
-            pass
+    return None
```

Comment:
redundant return

Reply:
expected by mypy, please refer to reply above